File: scenarios/vb-material-theme-darker/input.vb
Imports System

Module Demo
    ' Compute factorial recursively
    Function Factorial(ByVal n As Integer) As Integer
        If n <= 1 Then
            Return 1
        End If
        Return n * Factorial(n - 1)
    End Function

    Sub Main()
        Dim name As String = "World"
        Dim numbers() As Integer = {1, 2, 3, 4, 5}
        Dim total As Integer = 0

        For Each n As Integer In numbers
            total += n
        Next

        Console.WriteLine($"Hello, {name}!")
        Console.WriteLine("5! = " & Factorial(5))
    End Sub
End Module

File: scenarios/clojure-nord/input.clj
(ns demo.core
  (:require [clojure.string :as str]))

;; Compute factorial recursively
(defn factorial [n]
  (if (<= n 1)
    1
    (* n (factorial (dec n)))))

(def greeting "Hello, world!")

(defn greet [name]
  (str/join " " ["Hi" name "you are number" (factorial 5)]))

(let [nums [1 2 3 4 5]
      total (reduce + 0 nums)]
  (println greeting)
  (println (map #(* % %) nums))
  {:total total :keyword :ok :ratio 3/4})
